docs/site/development/development-process.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Handling Tickets (bugs/enhancements)
+---
+
+# Handling Tickets (bugs/enhancements)
+
+### Filing tickets
+
+  1.  You can add set other fields, but don't change the Owner. That way the committee sees the incoming tickets (those marked as Owner: somebody).
+        
```

Comment:
There are extra blank lines between list items; these should be removed.

Reply:
Also, in the Summary, add a link to the comparison between the old (in plaintext) and the new (in plaintext).